### PR TITLE
Added config file.

### PR DESCRIPTION
**Pull request by tblanshard**
Added a config file to store the project ID and the service account name.

@andreeanica16 @iht - thank you :)

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <head> 
         <meta charset="UTF-8">
         <title>Dataflow Data Analyser</title>
+        <script id="configFile" type="text/javascript" src="config.js"></script>
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
```
Comment (andreeanica16) [inline src/main/webapp/index.html:8]: Maybe you could group all the script tags to be easier to be located.
```suggestion
        <link rel="stylesheet" href="style.css">
        <script id="configFile" type="text/javascript" src="config.js"></script>
        <script src="script.js"></script>
```

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -5,16 +5,13 @@
...
-
 function initBody() {
     fetch('/data').then(console.log('Works'));
-    projectID = "bt-dataflow-sql-demo";
     checkPermissions();
 }
 
 function checkPermissions() {
-    fetch('/check-permissions?projID='+projectID, {method:"GET"})
+    fetch('/check-permissions?projID='+config.projectID+'&saID='+config.serviceAccount, {method:"GET"})
```
Comment (andreeanica16) [inline src/main/webapp/script.js:14]: fetch does a GET request by default, when no options are specified. You could write simplier
```suggestion
    fetch('/check-permissions?projID='+config.projectID+'&saID='+config.serviceAccount)
```

Comment (tblanshard) [reply src/main/webapp/script.js:14]: Ah good point - I only started putting GET in as well as POST to make things easier to read when looking for different functions but I agree that it's simpler without them :)

File: src/main/webapp/index.html (hunk 1 above)
Comment (tblanshard) [reply src/main/webapp/index.html]: Yes that's definitely neater!

**Pull request merged**